build_tools/cmake/iree_static_linker_test.cmake

```diff
@@ -0,0 +1,259 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include(CMakeParseArguments)
+
+# iree_static_linker_test()
+#
+# Creates a test with statically linked libraries. Assuming the target backend
+# is llvm-cpu and the driver is the local-sync driver.
```

Comment:
Let's expand this comment to explain what the generated test covers, and reword the "Assuming the..." to something more like "test for the statically linked libraries generated by the llvm-cpu compiler target backend and executed using the local-sync runtime HAL driver." As only the llvm-cpu target generates static libraries, that's a more direct way to explain it. With the other types of tests, a generic description with exclusions makes sense ("compile and run your program on a supported backend").

Reply:
Done.